src/attribute.ts

```diff
@@ -26,13 +29,17 @@ export type AttributeOptions = Partial<{
   name: string
   type: () => any
   persist: boolean
+  dirtyChecker?: DirtyChecker<any>
 }>
 
+export const STRICT_EQUALITY_DIRTY_CHECKER: DirtyChecker<any> = (prior, current) => (prior !== current)
```

Comment:
Since lodash is already a dependency, we may use [`_.isEqual()`](https://lodash.com/docs/4.17.15#isEqual) as default dirty checker